spec/controllers/gradings_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe GradingsController do
+  describe 'GET new' do
+    before { get :new }
+
+    it 'assigns @summary' do
+      expect(assigns(:summary)).to be_a(Hash)
+      expect(assigns(:summary)[:qn_ans]).to be_a(Hash)
+    end
+
+    it 'exposes questions' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.